source/_patterns/01-molecules/teasers/teaser.yaml

```diff
@@ -89,13 +89,14 @@ schema:
       properties:
         meta:
           $ref: ../../01-molecules/components/meta.yaml#/schema
-        publishState:
+        formats:
           type: object
+          minProperties: 1
           properties:
-            vor:
+            html:
```

Comment:
new format is more extensible, as new keys can be added without changing the existing ones